### ISSUE TITLE
[MU4 Issue] MSBasic "Warm Synthesizer" is erroneously mapped to "String" sound

### ISSUE THREAD

**Issue by ShawTakeuchi** [musescore/MuseScore#15236]
**Describe the bug**

MS4 MS-Basic "Warm Synthesizer" instrument is erroneously mapped to "String" 

**To Reproduce**

Open a new score, select Electronics->Warm Synthesizer and play.

**Platform information**

Win10/64, MuseScore 4.0 (MuseHub not instsalled)

**Additional context**

See (and hear) attached .zip for .mscz and two MP3 audio files for comparing MS3 vs MS4.

[MS4-WarmSynth-MS3vsMs4.zip](https://github.com/musescore/MuseScore/files/10248460/MS4-WarmSynth-MS3vsMs4.zip)

**Comment by chrisgibbs44** [comment]
Yeah I've noticed this also, still occurring in MuseScore 4.0.2

**Comment by Jojo-Schmitz** [comment]
So it is a regression

**Comment by ShawTakeuchi** [comment]
@chrisgibbs44 

While waiting for the devs to tackle this issue, Marc Sabatella has a good workaround:
https://community.masteringmusescore.com/c/resources/ms-basic-individual-sounds

**Comment by vashutangri** [comment]
I could try working on this issue

**Comment by cbjeukendrup** [comment]
@vashutangri One hint: you can use this program to inspect the sounds in the SoundFont: https://www.polyphone-soundfonts.com
This program lists the sounds, so that you can see what's the MIDI bank and program for the correct sound.

**Comment by switerom** [comment]
I explored FluidR3Mono_GM.sf3 and "Warm Synth" really assigned incorrectly. 
![3](https://user-images.githubusercontent.com/102316983/233266823-5a3f3b9c-d221-4638-bd9e-1c63b1a7538d.png)
But the thing is, there just aren't any samples that sound like "Warm Synth Pad". 
In MuseScore 2 the same file has other samples as
![4](https://user-images.githubusercontent.com/102316983/233266902-05722077-75e4-45a8-aefd-d90a9c4b5c5f.png)
signed, but they sound also like usual strings.

**Comment by cbjeukendrup** [comment]
But the sound font used by MuseScore is MS Basic, not FluidR3Mono.

**Comment by ShawTakeuchi** [comment]
> But the sound font used by MuseScore is MS Basic, not FluidR3Mono.

Investigating this further inside the sf3 file(s), looks like "Warm Pad" is missing the waveforms (ogg) from "MS Basic.sf3" that is included in Musescore 4.0, 4.1.

The original MS3.x FluidR3Mono_GM.sf3 has this:

![ms36](https://github.com/musescore/MuseScore/assets/116447844/adc24148-2c9d-4450-96e7-70f9aa2306f1)

In the MS4.x "MS Basic.sf3" waveforms are missing from "Warm Pad" and linked to "Strings"

![ms41](https://github.com/musescore/MuseScore/assets/116447844/e60faebe-d5cd-4396-aae0-77a715b328df)

Furthermore looks like Warmpad ogg files are completely missing from the list in MS Basic.sf3. Perhaps it got left out during the conversion process, and the compacting tool defaulted to "strings" due to an error?  In playback audio it does sound like strings.

Could someone explain why FluidR3Mono_GM.sf3 is in MS4.x next to "MS Basic.sf3"?  Is there an overriding relationship?

Side note:  Warm Pad is useful/important for doing Ethereal genre compositions, or in quiet passages to back solo instruments in cinematic music with heavy reverb.  I also used it for "subliminal fill" in fusion jazz!

**Comment by cosmoglint** [comment]
Hi Folks, is this issue available for claiming? I would like to work on this issue

**Comment by cbjeukendrup** [comment]
Yes, you're welcome! But as mentioned above, it turns out that this is a problem inside the SoundFont file, rather than in any kind of code file, so keep that in mind.

**Comment by Declan-Griner** [comment]
I'm pretty comfortable using Polyphone and editing SoundFont files. I'd like to work on this

**Comment by cbjeukendrup** [comment]
@Declan-Griner That would be great, thanks!

**Comment by dwlakes** [comment]
Looks like this issue still hasn't been fixed? I just tried it on 4.2.1 and it still has the string sound. Could I try fixing it?

**Comment by cbjeukendrup** [comment]
@dwlakes I wouldn't recommend working on this issue, because it involves editing the soundfont itself rather than code. Since the soundfont is a binary file, we'd rather edit it ourselves, because otherwise we have no way of figuring out what's actually changed. It's not that we are distrustful of contributors, but just that we like to know what's in our codebase :)

**Comment by mrbumpy409** [comment]
The problem is that the lowpass filter in MuseScore's FluidSynth build is no longer working, which messes with the intended sound of many of MS_Basic's presets, for example where acoustic instruments are supposed to become more mellow at low velocities. Now, everything that is supposed to use the lowpass filter to sound mellow or dark (such as "Warm Pad"), now always plays the samples at full brightness.
